example/src/Examples/API/Data.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { StyleSheet, Dimensions } from "react-native";
+import {
+  AlphaType,
+  Canvas,
+  ColorType,
+  Image,
+  Skia,
+} from "@shopify/react-native-skia";
+
+const { width } = Dimensions.get("window");
+const SIZE = width;
+
+const pixels = new Uint8Array(256 * 256 * 4);
+pixels.fill(255);
```

Comment:
A suggestion - since this results in a white image (and might be mistaken for a bug), what about doing something like this:

```typescript
let i = 0;
for (let x = 0; x < 256 * 4; x++) {
  for (let y = 0; y < 256 * 4; y++) {
    pixels[i++] = (x * y) % 255;
  }
}
```

Resulting in an image like this:

<img width="196" alt="image" src="https://user-images.githubusercontent.com/875252/155715434-4a8a880c-430e-45eb-9d25-969a2b0c6314.png">